doc/manual/manual.tex

```diff
@@ -1315,14 +1316,233 @@ \subsection{Numerical methods}
+there are a number of pitfalls if one wanted to use them in \aspect{}. We will
+discuss these in the following. Since they are typically only provided
+considering velocity, pressure and temperature, we will in the following omit
+the dependence on the compositional fields used in previous sections, though
+this dependence can easily be added back into the equations stated below. A
+detailed discussion of the approximations introduced below can also be found in
+\cite{KLKLZTTK10}.
+
+The three approximations mentioned all start by writing the pressure and
+temperature as the sum of a (possibly depth dependent)
+reference value plus a perturbation, i.e., we will write
+\begin{align*}
+  p(\mathbf x,t) &= \bar p(z) + p'(\mathbf x,t),
+  \\
+  T(\mathbf x,t) &= \bar T(z) + T'(\mathbf x,t).
+\end{align*}
+Here, barred quantities are reference states and may depend on the depth $z$
+(not necessarily the third component of $\mathbf x$) whereas primed quantities
+are the spatially and temporally variable deviations of the temperature and
+pressure fields from this reference state. In particular, the reference pressure
+is given by solving the hydrostatic equation,
+\begin{align*}
+  \nabla \bar p = \bar\rho \mathbf g,
+\end{align*}
+where $\bar\rho=\rho(\bar p,\bar T)$ is a \textit{reference density} that
+depends on depth. $\bar T$ is chosen as an adiabatic profile accounting for the
+fact that the temperature increases as the pressure increases, but also taking
+\marginpar{Scott: I don't see this discussed in \cite{KLKLZTTK10}, but one
+needs to make $\bar T$ also take into account diffusion so that it disappears
+from the temperature equation in the ALA.} 
+into account thermal diffusion effects. It also depends on depth.
+With these definitions, equations \eqref{eq:stokes-1}--\eqref{eq:stokes-2} can equivalently be written as follows:
+\begin{align}
+  \label{eq:stokes-decomposed-1}
+  -\nabla \cdot \left[2\eta \left(\varepsilon(\mathbf u)
+                                  - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+                \right] + \nabla p' &=
+  (\rho-\bar\rho) \mathbf g
+  & \qquad
+  & \textrm{in $\Omega$},
+  \\
+  \label{eq:stokes-decomposed-2}
+  \nabla \cdot (\rho \mathbf u) &= 0
+  & \qquad
+  & \textrm{in $\Omega$}.
+\end{align}
+Likewise, when omitting entropic effects, the temperature equation reads as
+\begin{multline}
+  \label{eq:temperature-decomposed}
+  \rho C_p \left(\frac{\partial T}{\partial t} + \mathbf u\cdot\nabla T\right)
+  - \nabla\cdot k\nabla T
+  \\
+  =
+  \rho H
+  +
+  2\eta
+  \left(\varepsilon(\mathbf u) - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+  :
+  \left(\varepsilon(\mathbf u) - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+  +\alpha T \left( \mathbf u \cdot \nabla p \right)
+  \quad
+  \textrm{in $\Omega$}.
+\end{multline}
+
+Starting from these equations, the approximations discussed in the next few
```

Comment:
You've already used "starting from" in the previous paragraph. Maybe "Based on these equations, ..."